Extension/src/nativeStrings.json

```diff
@@ -403,8 +403,8 @@
     },
     "refactor_create_declaration_definition_failed":
     {
-        "text": "Create Declaration / Definition failed:",
-        "hint": "The operation 'Create Declaration / Definition' on a function was not successfully."
+        "text": "Create Declaration / Definition failed: %s",
+        "hint": "The operation 'Create Declaration / Definition' on a function was not successful. %s is the info why it failed."
```

Comment:
Should you mention that %s ends with a period in case a translator incorrectly adds another one?

Another alternative would be to revert that change and it keep it ending with ":". I didn't realize the %s ended in a period previously.

Reply:
By having the %s it shows additional info about the failure is expected by the caller of the string "Create Declaration / Definition failed:".